swift/ios-shoppe-demo/ios-shoppe-demo/ShoppeItem_BACKUP_73793.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  ShoppeItem.swift
+//  ios-shoppe-demo
+//
+//  Created on 3/4/20.
+//  Copyright © 2020 FullStory All rights reserved.
+//
+
```

Comment:
It looks like there might still be some residual issues from a prior merge:

![image](https://user-images.githubusercontent.com/45576380/76275130-e06b6480-6258-11ea-9041-31cc5f2e2e48.png)